src/js/jsx/sections/libraries/LibraryBar.jsx

```diff
@@ -27,21 +27,30 @@ define(function (require, exports, module) {
 
     var React = require("react"),
         Fluxxor = require("fluxxor"),
-        FluxMixin = Fluxxor.FluxMixin(React);
+        FluxMixin = Fluxxor.FluxMixin(React),
+        Immutable = require("immutable"),
+        strings = require("i18n!nls/strings");
```

Comment:
strings should be part of the require group below, as we go with the convention of: External, Adapter, Internal